x-pack/plugins/stack_connectors/public/connector_types/jira/jira_params.tsx

```diff
@@ -424,7 +424,7 @@ const JiraParamsFields: React.FunctionComponent<ActionParamsProps<JiraActionPara
               </>
             }
             onDocumentsChange={(json: string) => {
-              editSubActionProperty('otherFields', json);
+              editSubActionProperty('otherFields', json === '' ? null : json);
```

Comment:
Can `json == '{}'`? Should we fall back to `null` in that case as well?

Reply:
yeah but that value is valid so it would be fine. I didn't want the users to type `{}` in the box and then the value disappears because we fall back to `null` 😄